lib/ramble/ramble/cmd/workspace.py

```diff
@@ -355,6 +355,12 @@ def workspace_setup_setup_parser(subparser):
              'all scripts. Prints commands that would be executed ' +
              'for installation, and files that would be downloaded.')
 
+    subparser.add_argument(
+        '--n_repeats', dest='n_repeats',
+        nargs='+',
```

Comment:
Why is this + not 1? What happens if we give multiple args here?

Reply:
That's a mistake, it should only accept 1 arg.